var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py

```diff
@@ -66,8 +72,10 @@ class PyPennylaneLightningKokkos(CMakePackage, PythonExtension, CudaPackage, ROC
     depends_on("py-pybind11", type="build")
     depends_on("py-pip", type="build")
     depends_on("py-wheel", type="build")
-    depends_on("py-pennylane@0.28:", type=("build", "run"))
-    depends_on("py-pennylane-lightning@0.28:~kokkos", type=("build", "run"))
+    depends_on("py-pennylane@:0.29", type=("build", "run"), when="@0.29")
+    depends_on("py-pennylane@0.30:", type=("build", "run"), when="@0.30:")
```

Comment:
It says 0.28+ in the setup.py

Reply:
Good catch, yes this should be 0.28. I'll make an update.